keps/349-all-or-nothing/README.md

```diff
@@ -225,6 +246,28 @@ type WaitForPodsReady struct {
 	// RequeuingStrategy defines the strategy for requeuing a Workload.
 	// +optional
 	RequeuingStrategy *RequeuingStrategy `json:"requeuingStrategy,omitempty"`
+
+	// WaitForAdmissionChecks defines configuration for the WaitForAdmissionChecks feature,
+	// which is used to ensure that specific AdmissionCheck is ready within the specified time.
+	// If there is no specified configuration for an AdmissionCheck, there is no timeout for it.
+	// +listType=map
+	WaitForAdmissionChecks *WaitForAdmissionChecks
```

Comment:
I think this needs to be a list so that we can pose the timeout for multiple admission checks.

Reply:
Indeed, that's what I meant. Thanks for point this out, done!